sci-misc/boinc/boinc-7.9.3.ebuild

```diff
@@ -0,0 +1,196 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+WX_GTK_VER=3.0-gtk3
+
+inherit autotools eutils gnome2-utils linux-info systemd user versionator wxwidgets
+
+MY_PV="7.10"
+
+DESCRIPTION="The Berkeley Open Infrastructure for Network Computing"
+HOMEPAGE="http://boinc.ssl.berkeley.edu/"
+SRC_URI="https://github.com/BOINC/boinc/archive/client_release/${MY_PV}/${PV}.tar.gz -> ${P}.tar.gz
+	X? ( http://boinc.berkeley.edu/logo/boinc_glossy2_512_F.tif -> ${PN}.tif )"
+RESTRICT="mirror"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="X cuda curl_ssl_gnutls curl_ssl_libressl +curl_ssl_openssl static-libs"
```

Comment:
USE `static-libs` is nowhere used. `--disable-static` is passed in `src_configure`, but I definitely saw `.a` getting installed yesterday. It seems that build flag does not work, can you check?

Reply:
Well, this seems to be another leftover from the past. I am currently testing what happens if both --enable-dynamic-client-linkage and --enable-static depend on USE="static-libs".

Reply:
Okay, with USE="-static-libs" the ebuilds install libboinc.a and libboinc_crypt.a. Both are "static-only", so that is okay. If their installation would be depending on the static-libs USE flag, applications using them would have to force that flag on boinc.

The static-libs flag is now heeded and installs static versions of the other libraries.
However, should the installation of dynamic libraries then be turned off? Currently both are installed.

Reply:
Please do not turn dynlibs off, see global `static-libs` description: `Build static versions of dynamic libraries as well`

Personally I don't care about static libs, so I would just drop the currently no-op static-libs flag and maintain the --disable-static build switch was added for a good reason in the past.

Reply:
Agreed. I haven't seen any need for static libs anywhere anyway. If, for some weird reason, a demand arises, that flag can be re-added.